<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.example.fourdsight.service.ImageServiceTest" tests="3" skipped="0" failures="0" errors="0" timestamp="2018-09-12T13:50:35" hostname="EVT01279NB" time="0.045">
  <properties/>
  <testcase name="shouldSaveImage" classname="com.example.fourdsight.service.ImageServiceTest" time="0.004"/>
  <testcase name="shouldSendMessageWhenUrlListIsValid" classname="com.example.fourdsight.service.ImageServiceTest" time="0.02"/>
  <testcase name="shouldThrowExceptionWhenUrlListIsInvalid" classname="com.example.fourdsight.service.ImageServiceTest" time="0.014"/>
  <system-out><![CDATA[16:50:35.519 [Test worker] ERROR com.example.fourdsight.service.UrlValidatorService - Invalid url with exception message :no protocol: invalidUrl
]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
